ansible/pbTestScripts/qemu_test_script.sh

```diff
@@ -139,24 +133,27 @@ done
 		"S390X" )
 			export MACHINE="s390-ccw-virtio";
 			export DRIVE="-drive file=$workFolder/S390X.dsk,if=none,id=hd0 -device virtio-blk-ccw,drive=hd0,id=virtio-disk0";
-			export COMMAND="s390x";;
```

Comment:
Can we change the `COMMAND` variable to be `QEMUARCH` or something like that? Since it's not the full command name it's a bit misleading.

Reply:
Change made 👍